src/beast/beast/utility/Journal.h

```diff
@@ -107,76 +123,98 @@ class Journal
 
     class Stream;
 
-    /** Scoped ostream-based container for writing messages to a Journal. */
+private:
+    /* Scoped ostream-based container for writing messages to a Journal. */
```

Comment:
Not in your change set. I'll note that `Journal` has three nested classes: `ScopedStream`, `Stream`, and  `Sink`. It's easy to get lost in the definition of `Journal` with three largish nested classes. I'd rather have them in a detail namespace or somesuch (no change needed).